### PR TITLE
Add support for phpspec ^6.1

### PR DESCRIPTION
**Pull request by dannyvw**

### CODE REVIEW

**Comment by Nyholm** [comment]
Maybe we should stop testing on PHP 7.1 and include PHP7.4 instead?

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "phpspec/phpspec": "^5.0"
+        "phpspec/phpspec": "^6.1"
```
Comment (GrahamCampbell) [inline composer.json:20]: I think this should be `^5.0|^6.0`. Why drop 5? If we are going to drop 5, then we probably want another major release, like in the past.

**Comment by GrahamCampbell** [comment]
> Maybe we should stop testing on PHP 7.1 and include PHP7.4 instead?

Only if 5 is actually going to be dropped, rather than going with my suggestion of `^5.0|^6.0` for the version.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
See comments.

**Comment by dannyvw** [comment]
See https://github.com/akeneo/PhpSpecSkipExampleExtension/pull/23

**Pull request closed without merging**